lib/src/dbs/options.rs

```diff
@@ -14,9 +14,9 @@ use std::sync::Arc;
 #[derive(Clone, Debug, Eq, PartialEq)]
 pub struct Options {
 	// Currently selected NS
-	pub ns: Option<Arc<String>>,
+	pub ns: Option<Arc<str>>,
 	// Currently selected DB
-	pub db: Option<Arc<String>>,
+	pub db: Option<Arc<str>>,
```

Comment:
Same as with the `Vec`'s (unless you plan to extract the `String` when the reference count drops to 1, `Arc<str>` is less indirected than `Arc<String>`)